examples/jest-vtu-example/.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "env": {
+    "test": {
+      "presets": ["env", "vue-app"]
```

Comment:
isnt it presets for babel 6?  Tthere could be inconsistencies between nuxt vs jest then since nuxt use babel 7. 
Maybe @babel/preset-env \ @nuxtjs/babel-preset-app  although it need checking